source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -302,14 +306,24 @@ export class StringPropertyEditor extends SingleInputPropertyEditor {
             return [
                 {
                     caption: "...",
+                    altText: (this.label + " " + "Data Binding"),
+                    expanded: false,
                     onExecute: (sender: SingleInputPropertyEditor, clickedElement: HTMLElement) => {
+                                               
+                        clickedElement.setAttribute("aria-label", this.label + " " + "Data Binding");
```

Comment:
If I understand correctly, the above altText gets passed to action.accessibleTitle, which gets rendered by the action. Do we need to set it again?

Reply:
You're right, this is a remnant of a previous version of this change where i had the "Expanded" and "Collapsed" in the label and therefore had to change them on expand collapse. I can send a PR to fix this.

---
In reply to: [583022680](https://github.com/microsoft/AdaptiveCards/pull/5419#discussion_r583022680) [](ancestors = 583022680)